subprojects/junit-jupiter/src/test/java/org/mockitousage/CloseOnDemandTest.java

```diff
@@ -0,0 +1,37 @@
+package org.mockitousage;
```

Comment:
This file is missing a license header:

```
/*
 * Copyright (c) 2021 Mockito contributors
 * This program is made available under the terms of the MIT License.
 */
```

You can run `./gradlew build` locally to get it passing. (You might also need to run `./gradlew spotlessApply` to make the formatter happy)